internal/clients/embed.go

```diff
@@ -0,0 +1,54 @@
+package clients
```

Comment:
I'd move this one in `ec2` as well :thining:

Reply:
I commented already below, why?

I was thinking we would have a common type with a common function(s) for all types. And a single YAML with all the lists. Why you think this should go into ec2 again?

Reply:
Oh I get it, keeping only ec2 related functions there and common code in clients. Yeah, that makes sense!